### DISCUSSION TITLE
AI model help

### DISCUSSION THREAD

**Discussion by zoroxide** [community/community#124416] [Programming Help]
### Body

What about an AI model that recommends food recipes?
training using PyTorch or TensorFlow

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by shahd-omer** [comment]
Building a recipe recommendation AI involves data collection, preprocessing, model selection, training, evaluation, and deployment. Whether you use PyTorch or TensorFlow, focus on creating a model that understands ingredient combinations and can suggest delicious recipes based on user inputs. Happy coding

**Comment by ebndev** [comment]
Hi @zoroxide & @shahd-omer, thanks for being a part of the GitHub Community!

Please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
